src/factories/document.factory.ts

```diff
@@ -26,30 +26,31 @@ import {Oas30Document} from "../models/3.0/document.model";
  */
 export class OasDocumentFactory {
 
+    private V2_DEFAULT_MINOR = "0";
+    private V3_DEFAULT_MINOR = "0";
+    private V3_DEFAULT_PATCH = "2";
+
     /**
      * Creates a new, empty instance of an OAS document.
      * @param oasVersion
      * @return {OasDocument}
      */
     public createEmpty(oasVersion: string): OasDocument {
-        if (typeof oasVersion === "number") {
-            oasVersion = "" + oasVersion;
-        }
+        const ver = oasVersion;
```

Comment:
Add a type to `ver`... probably should be `string` based on the input param type, but could also perhaps be `string|number`.  Dealer's choice. :)  I just like having types for things when possible (not always possible).